client/src/components/navbar/index.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  AppBar,
+  Chip,
+  Container,
+  Menu,
+  MenuItem,
+  Toolbar,
+} from '@mui/material';
+import AccountCircleIcon from '@mui/icons-material/AccountCircle';
+import { useState, MouseEvent } from 'react';
+import { styles } from '@/components/navbar/styles';
+
+const Navbar = () => {
+  const [username] = useState('Test');
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const open = Boolean(anchorEl);
+  const handleClick = (event: MouseEvent<HTMLDivElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+  // TODO: Write store integration to actually logout user
+  const handleLogout = () => alert('Logout!');
+
+  return (
+    <AppBar
+      color="inherit"
+      position="sticky"
+      sx={{ mb: '1rem', boxShadow: 'none' }}
+    >
+      <Container maxWidth="xl">
+        <Toolbar disableGutters sx={styles.toolbar}>
+          <Chip
+            icon={<AccountCircleIcon />}
```

Comment:
I guess you can use avatar prop instead https://mui.com/material-ui/react-chip/#avatar-chip